internal/discovery/foreign-cluster-operator/foreign-cluster-operator_test.go

```diff
@@ -642,4 +636,245 @@ var _ = Describe("ForeignClusterOperator", func() {
 
 	})
 
+	Context("Test Permission", func() {
+
+		const (
+			outgoingBinding = "liqo-binding-liqo-outgoing"
+			incomingBinding = "liqo-binding-liqo-incoming"
+		)
+
+		var (
+			clusterRole1 rbacv1.ClusterRole
+			clusterRole2 rbacv1.ClusterRole
+		)
+
+		type permissionTestcase struct {
+			fc               discoveryv1alpha1.ForeignCluster
+			expectedOutgoing types.GomegaMatcher
+			expectedIncoming types.GomegaMatcher
+		}
+
+		JustBeforeEach(func() {
+			clusterRole1 = rbacv1.ClusterRole{
+				ObjectMeta: metav1.ObjectMeta{
+					Name: "liqo-outgoing",
+				},
+			}
+			clusterRole2 = rbacv1.ClusterRole{
+				ObjectMeta: metav1.ObjectMeta{
+					Name: "liqo-incoming",
+				},
+			}
+
+			Expect(controller.Client.Create(ctx, &clusterRole1)).ToNot(HaveOccurred())
```

Comment:
```suggestion
			Expect(controller.Client.Create(ctx, &clusterRole1)).To(Succeed())
```

Nit. Just a bit more self explanatory (and shorter)